### ISSUE TITLE
Pre-test software?

### ISSUE THREAD

**Issue by martin9115** [shayanalibhatti/Finger-Movement-Classification-via-Machine-Learning-using-EMG-Armband-for-3D-Printed-Robotic-Hand#1]
Hi, great project.
Did you use some pre-test software to draw the emg arm and test the program virtually, before print it ?

**Comment by shayanalibhatti** [comment]
No, i had to do a few iterations to get it all right. For design, i used DesignSpark Mechanical's 3d design software.

**Comment by martin9115** [comment]
Did you know any software/program, in which we can design and pre-test code emg arm virtually?

**Comment by shayanalibhatti** [comment]
No
